spec/features/conference_registration_spec.rb

```diff
@@ -49,5 +66,15 @@
         expect(conference.user_registered?(participant)).to be(true)
       end
     end
+
+    context 'registration is closed' do
+      let(:conference_with_closed_registration) { create(:conference, registration_period: create(:registration_period)) }
+
+      scenario 'registers for a conference', feature: true do
+        visit new_conference_conference_registrations_path(conference_with_closed_registration.short_title)
+        expect(current_path).to eq(root_path)
+        expect(flash).to eq 'You are not authorized to access this page.'
```

Comment:
> Why is this the message?

When registration is closed, request for new registration get caught by authorization of those requests. Thats why flash message is about denied access. I think we can customize error message, how closed registration is not the only time auth would fail. I guess we can be more generic:
```Ruby
authorize! :new, @registration, message: "You can't register to this conference"
```